### Задача 1

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя уровень значимости 0.025 и правостороннюю критическую область, проверить данную гипотезу при условии, что в выборке из 100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

In [1]:
import numpy as np

Задаём параметрs

In [2]:
n = 100
loc = 17.5
scale = 2

In [3]:
mu = 17 # среднее значение имеет такое же распределение как случайная величина
sigma = scale / np.sqrt(n) # согласно ЦПТ получаем стандартное отклонение
alpha = 0.025 # Заданный уровень значимости

In [4]:
(mu + 2 * sigma, float('inf')) # правосторонная область

(17.4, inf)

За функцию Т мы взяли среднее арифметическое. Среднее значение выборки из 100 входит в правостороннюю критическую область 17.5 > 17.4, поэтому нулевая гипотеза, что шарики для подшипников имеют средний диметр 17 мм отвергается в пользу альтернативной гипотезы, что средний димаетр не равен 17 мм 

### Задача 2

Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочное среднее равно 80, а объём равен 256.

In [5]:
scale = 16
loc = 80
n = 256

In [6]:
(loc - 2 * scale / np.sqrt(n), loc + 2 * scale / np.sqrt(n)) # искомый интервал

(78.0, 82.0)

### Задача 3

Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:

202, 203, 199, 197, 195, 201, 200, 204, 194, 190

Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [7]:
sample = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
mean = sample.mean()
std = sample.std(ddof=1)

In [8]:
t = 3.2498 # получаем коэффициент Стьюдента из таблицы

In [9]:
(mean - t * std / np.sqrt(n), mean + t * std / np.sqrt(n)) # Доверительный интервал

(197.59544598179602, 199.40455401820398)

Утверждение продавца неверно так как с вероятностью 99%  средний вес пачки находится в интервале

от 197.59544598179602 до 199.40455401820398